app/controllers/api/v1/deliveries_controller.rb

```diff
@@ -13,10 +35,13 @@ def create
       end
     end
 
+    api :GET, '/v1/deliveries/1', "Get a delivery"
     def show
       render json: @delivery, serializer: serializer
     end
 
+    api :PUT, '/v1/deliveries/1', "Update a delivery"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.